docs/core/allocation.qmd

```diff
@@ -5,16 +5,26 @@ title: "Allocation"
 # Introduction
 Allocation is the process of assigning an allocated flow rate to demand nodes in the physical layer of the model based on information about sources, the different demand nodes over various priorities, constraints introduced by nodes, local water availability and graph topology. The allocation procedure implemented in Ribasim is heavily inspired by the [maximum flow problem](https://en.wikipedia.org/wiki/Maximum_flow_problem).
 
-The allocation problem is solved per subnetwork of the Ribasim model. The subnetwork is used to formulate an optimization problem with the [JuMP](https://jump.dev/JuMP.jl/stable/) package, which is solved using the [HiGHS solver](https://highs.dev/). For more in-depth information see also the example of solving the maximum flow problem with `JuMP.jl` [here](https://jump.dev/JuMP.jl/stable/tutorials/linear/network_flows/#The-max-flow-problem).
+The allocation problem is solved per subnetwork (and main network) of the Ribasim model. Each subnetwork is used to formulate an optimization problem with the [JuMP](https://jump.dev/JuMP.jl/stable/) package, which is solved using the [HiGHS solver](https://highs.dev/). For more in-depth information see also the example of solving the maximum flow problem with `JuMP.jl` [here](https://jump.dev/JuMP.jl/stable/tutorials/linear/network_flows/#The-max-flow-problem).
+
+:::{.callout-note}
+within this *Allocation* section the main network is also considered to be a subnetwork.
+:::
 
 # The high level algorithm
-The allocation algorithm consists of 3 types of optimization problems:
+The allocation algorithm contains 3 types of optimization:
 
-1. **Subnetwork demand collection**: Collect the demands of a subnetwork from the main network by optimizing with unlimited capacity from the main network;
-2. **Main network allocation**: Allocate to subnetworks with the above collected demand, and demands in the main network;
-3. **Subnetwork allocation**: Allocate to demands in the subnetworks with the flows allocated to the subnetwork in the main network allocation.
+- `internal_sources`, where flows are allocated within a subnetwork by only using sources inside the subnetwork;
+-  `collect_demands`, where flows are allocated within a subnetwork by only using the main network inlet as a source, with demands reduced by allocations in `internal_sources`;
```

Comment:
There is a double space after `-`. Also this phrasing is a bit confusing to me, since the allocated flows are not used, this step is only used to determine demands at subnetwork sources. Perhaps it's good to add this.